test/integration/api/projects.js

```diff
@@ -1219,8 +1235,7 @@ describe('api: /projects?forms=true', () => {
           body.length.should.equal(1);
           body[0].should.be.a.Project();
           const { formList, verbs } = body[0];
-          // verbs seems to have 1 more verb than it should - one is duplicated
-          verbs.length.should.be.greaterThan(40);
+          verbs.length.should.equal(40); // REVIEWER: master currently returns duplicate of 'submission.update'
```

Comment:
It seems like a nice effect of `array_agg(distinct)` that the duplicated verb is no longer returned over the API. (Maybe at some point we should also remove it from the database. 🤔) Can the REVIEWER comment be removed now?